src/libraries/Lockers.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.20;
+
+/// @notice This is a temporary library that allows us to use transient storage (tstore/tload)
+/// for the lockers array and nonzero delta count.
```

Comment:
would help to make explicit TODO that we should remove this once solidity supports transience natively